src/TestLogger/Core/LoggerConfiguration.cs

```diff
@@ -16,6 +16,7 @@ public class LoggerConfiguration
     {
         public const string LogFilePathKey = "LogFilePath";
         public const string LogFileNameKey = "LogFileName";
+        public const string ParserKey = "Parser";
```

Comment:
We need to update our wiki https://github.com/spekt/testlogger/wiki/Logger-Configuration once this is merged.

Reply:
Yeah, i will take care of it